deploy-manage/tools/snapshot-and-restore/s3-repository.md

```diff
@@ -77,6 +77,9 @@ Define the relevant secure settings in each node’s keystore before starting th
 
 The following list contains the available client settings. Those that must be stored in the keystore are marked as "secure" and are **reloadable**; the other settings belong in the [`elasticsearch.yml`](/deploy-manage/stack-settings.md) file.
 
+`region`
+:   Specifies the region to use. When configured this value will determine the signing region and regional endpoint to use, unless the endpoint is overridden via the `endpoint` setting. If not specified, we will attempt to determine the region automatically using the SDK.
```

Comment:
wording nit: "set" is simpler than "configured" and "specified" and it's better to be consistent whichever word we choose. Also "we" is Elasticsearch here.

```suggestion
:   Specifies the region to use. When set, determines the signing region and regional endpoint to use, unless the endpoint is overridden via the `endpoint` setting. If not set, {{es}} will attempt to determine the region automatically using the AWS SDK.
```